source/developers-guide/index.html

```diff
@@ -15,5 +15,6 @@
     <li><a href="{{ site.url }}/developers-guide/shopware-5-performance-for-devs/">Shopware 5 performance guide for developers</a></li>
     <li><a href="{{ site.url }}/developers-guide/password-encoder/">Shopware password encoder</a></li>
     <li><a href="{{ site.url }}/developers-guide/debugging/">Debugging shopware</a></li>
+    <li><a href="{{ site.url }}/developers-guide/shopware-5-plugin-license/">Shopware 5 license check in plugins</a></li>
```

Comment:
Please get rid of shopware-5 in the filename and so on. It's just "License check in plugins"